js/repeater-frozen.js

```diff
@@ -0,0 +1,534 @@
+				}
+			}
+
+			if (!$.isArray(items)) {
+				items = [items];
+			}
+
+			if (force === true || selectable === 'multi') {
+				l = items.length;
+			} else if (selectable) {
+				l = (items.length > 0) ? 1 : 0;
+			} else {
+				l = 0;
+			}
+
+			for (i = 0; i < l; i++) {
+				if (items[i].index !== undefined) {
+					$item = this.$canvas.find('.repeater-frozen table tbody tr:nth-child(' + (items[i].index + 1) + ')');
+					if ($item.length > 0) {
+						selectItem($item, items[i].selected);
+					}
+
+				} else if (items[i].property !== undefined && items[i].value !== undefined) {
+					this.$canvas.find('.repeater-frozen table tbody tr').each(checkIfItemMatchesValue);
+				}
+
+			}
+		};
+
+		$.fn.repeater.Constructor.prototype.frozen_setFrozenColumns = function () {
+			var frozenTable = this.$canvas.find('.table-frozen');
+			var $table = this.$element.find('.repeater-frozen table');
+			var repeaterWrapper = this.$element.find('.repeater-frozen');
+
+			if (frozenTable.length < 1) {
+				//setup frozen column markup
+				//main wrapper and remove unneeded columns
+				var $frozenColumnWrapper = $('<div class="frozen-column-wrapper"></div>').insertBefore($table);
+				var $frozenColumn = $table.clone().addClass('table-frozen');
+				$frozenColumn.find('th:not(:first-child),td:not(:first-child)').remove();
+
+				//need to set absolute heading for vertical scrolling
+				var $frozenThead = $frozenColumn.clone().removeClass('table-frozen');
+				$frozenThead.find('tbody').remove();
+				var $frozenTheadWrapper = $('<div class="frozen-thead-wrapper"></div>').append($frozenThead);
+
+				$frozenColumnWrapper.append($frozenColumn);
+				repeaterWrapper.append($frozenTheadWrapper);
+			}
+
+			this.$element.find('.repeater-frozen table.table-frozen tr').each(function (i, elem) {
+				$(this).height($table.find('tr:eq(' + i + ')').height());
+			});
+			var columnWidth = $table.find('td:eq(0)').outerWidth();
+			this.$element.find('.frozen-column-wrapper, .frozen-thead-wrapper').width(columnWidth);
+
+		};
+
+		//ADDITIONAL DEFAULT OPTIONS
+		$.fn.repeater.defaults = $.extend({}, $.fn.repeater.defaults, {
+			frozen_columnRendered: null,
+			frozen_columnSizing: true,
+			frozen_highlightSortedColumn: false,
+			frozen_infiniteScroll: false,
+			frozen_noItemsHTML: 'no items found',
+			frozen_selectable: false,
+			frozen_sortClearing: false,
+			frozen_rowRendered: null,
+			frozen_frozenColumns: 0
+		});
+
+		//EXTENSION DEFINITION
+		$.fn.repeater.viewTypes.frozen = {
+			cleared: function () {
+			},
+			dataOptions: function (options) {
+				if (this.frozen_sortDirection) {
+					options.sortDirection = this.frozen_sortDirection;
+				}
+				if (this.frozen_sortProperty) {
+					options.sortProperty = this.frozen_sortProperty;
+				}
+				return options;
+			},
+			initialize: function (helpers, callback) {
+				this.frozen_sortDirection = null;
+				this.frozen_sortProperty = null;
+				callback();
+			},
+			resize: function () {
```

Comment:
This should be removed altogether unless something will reside within the function